recipes/rocksdb/all/conanfile.py

```diff
@@ -174,6 +178,9 @@ def package(self):
             self._remove_static_libraries()
             self._remove_cpp_headers() # Force stable ABI for shared libraries
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        pc_file = os.path.join(self.package_folder, "lib", "pkgconfig", "rocksdb.pc")
+        if os.path.exists(pc_file):
+            os.remove(pc_file)
```

Comment:
```suggestion
        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Just obliterate the pkgconfig in case of existing. It does not raise error when no having it installed.